src/analysis/complex/basic.lean

```diff
@@ -258,6 +258,17 @@ lemma isometry_of_real : isometry (coe : ℝ → ℂ) := of_real_li.isometry
 /-- Continuous linear map version of the canonical embedding of `ℝ` in `ℂ`. -/
 def of_real_clm : ℝ →L[ℝ] ℂ := of_real_li.to_continuous_linear_map
 
+/-- The only continuous ring homomorphisms from `ℂ` to `ℂ` are the identity and the complex
+conjugation. -/
+lemma ring_hom_eq_id_or_conj_of_continuous (f : ℂ →+* ℂ) (hf : continuous f) :
```

Comment:
I realize the location of this is a bit weird; it cuts in the middle of the definition `of_real_clm` and its API lemmas. Can you move it above `of_real_clm`? That's also the earliest it can go because `continuous_of_real` is used in the proof.

Reply:
Actually not even `continuous_of_real` is used; you can move it all the way up to just below the `normed_algebra R ℂ` instance if you want.

Reply:
What about putting it after ```continuous_conj```? After all, the result is a contrived way to prove that ```conj``` is continuous ;) 

A first version did indeed use ```continuous_of_real``` so that probably explains why I put it in that space...

Reply:
Sounds good. Grouping it with results about continuity is reasonable.

Reply:
Ok. Thanks.